api/api_full.go

```diff
@@ -742,6 +742,10 @@ type FullNode interface {
 
 	NodeStatus(ctx context.Context, inclChainStatus bool) (NodeStatus, error) //perm:read
 
+	// MethodGroup: Eth
+	// These methods are used for Ethereum-compatible JSON-RPC calls
+	EthBlockNumber(ctx context.Context) (string, error) //perm:read
```

Comment:
We should document how we're adapting this Ethereum call for Filecoin (i.e. in this case, returning the epoch of the heaviest tipset, i.e. nothing about blocks)